Control/Metrics/src/ArmoniKMeter.cs

```diff
@@ -35,155 +35,184 @@ namespace ArmoniK.Core.Control.Metrics;
 
 public class ArmoniKMeter : Meter, IHostedService
 {
-  private const    string                                                        QueuedName = "queued";
-  private readonly IDictionary<Tuple<string, string>, ObservableGauge<long>>     gauges_;
-  private readonly ILogger                                                       logger_;
-  private readonly ExecutionSingleizer<IDictionary<Tuple<string, string>, long>> measurements_;
-  private readonly ITaskTable                                                    taskTable_;
-  private          int                                                           i_;
+  private const string QueuedName = "queued";
+
+  private static readonly TaskStatus[] Statuses =
+  {
+    //TaskStatus.Unspecified,
+    TaskStatus.Creating,
+    TaskStatus.Submitted,
+    TaskStatus.Dispatched,
+    //TaskStatus.Completed,
+    //TaskStatus.Error,
+    //TaskStatus.Timeout,
+    //TaskStatus.Cancelling,
+    //TaskStatus.Cancelled,
+    TaskStatus.Processing,
+    TaskStatus.Processed,
+    TaskStatus.Retried,
+  };
```

Comment:
will it be possible to leave the choice at the user and having a configuration file or whatever which will provide the status that will be use in the metrics exporter?

Reply:
Yes, I've now added the code for it.